src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -336,7 +336,7 @@ void TwitchIrcServer::onMessageSendRequested(TwitchChannel *channel,
             if (this->lastErrorTimeSpeed_ + 30s < now)
             {
                 auto errorMessage =
-                    makeSystemMessage("sending messages too fast");
+                    makeSystemMessage("You are sending messages too fast.");
```

Comment:
```suggestion
                    makeSystemMessage("sending messages too quickly");
```

Reply:
My change was to make it into a sentence from a sentence fragment. However from "too fast" to "too quickly" sounds better.